pecos/xmc/xlinear/model.py

```diff
@@ -379,10 +381,22 @@ def predict(
         Returns:
             Y_pred (csr_matrix): prediction matrix
```

Comment:
Modify the docstring here: what are the prediction results when `select_outputs_csr` is given and not given.

Reply:
Docstring has been modified to specify different prediction results.